Library/Homebrew/brew.sh

```diff
@@ -349,7 +349,7 @@ then
   else
     # Used in ruby.sh.
     # shellcheck disable=SC2034
-    HOMEBREW_MACOS_SYSTEM_RUBY_NEW_ENOUGH="1"
+    export HOMEBREW_MACOS_SYSTEM_RUBY_NEW_ENOUGH="1"
```

Comment:
I wonder if we should export it in the `Declared in brew.sh` section below for consistency (that is, set it here, but export it there) and so that there is only one place to keep track of all exported variables.

Reply:
@maxim-belkin Good idea.

I also think, ideally, that all this `cleanup.rb` logic would go away and it'd just shell out to `ruby.sh` or similar instead so we can consolidate this logic in one place.

Reply:
(@maxim-belkin if you'd be able to pick that up I'd be very grateful ❤️)